data/json/npcs/NPC_Brigitte_LaCroix.json

```diff
@@ -87,7 +87,7 @@
     "gender": "female",
     "name_suffix": "Bone Seer",
     "class": "NC_BONE_SEER",
-    "attitude": 7,
+    "attitude": 1,
```

Comment:
This will make her go to the player to talk, did you check that it works? It was pretty cool to have her waiting hidden in the cemetery

Reply:
Hey - I did test it and it does work, but I get what you mean. Think I should put it back to 7? I felt it was more in line with her opening line asking you if you can hear the song.

Reply:
Bumped it back to 7. After taking a look at some videos, I agree her standing there feels... better.